gpu4pyscf/lib/gvhf-rys/rys_roots.cuh

```diff
@@ -416,11 +416,6 @@ __device__ static double ROOT_LARGEX_W_DATA[] = {
 5.2158612689910977e-01,
 };
 
```

Comment:
Please keep these size definitions

Reply:
These variables are defined at the beginning of the file. They are redefined here.